repository-data/webfiles/src/main/resources/site/freemarker/hstdefault/essentials-banner_nl.properties

```diff
@@ -0,0 +1,2 @@
+essentials-banner.ftl= Standaardtemplate
+without-description.ftl= Zonder beschrijving
```

Comment:
??

Reply:
We can get rid of this file, since its the dutch translation file